include/Math/Vector3D.hpp

```diff
@@ -7,6 +7,8 @@
 
 #pragma once
 
+#define IS_INVERSE(X,Y) (((X > 0 && Y < 0) || (X < 0 && Y > 0)) ? true : false)
```

Comment:
Change the name of the Macro to IS_ON_REAR

Reply:
Not agree